dashboard/app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -33,8 +37,27 @@ def all
     alias_method provider.to_sym, :all
   end
 
+  OAUTH_PARAMS_TO_STRIP = %w{oauth_token oauth_refresh_token}
+
+  def self.get_cache_key(oauth_param, user)
+    "#{oauth_param}_#{user.email}"
```

Comment:
I'll change this to a dash so that it at least doesn't conflict with the oauth params - it doesn't matter a lot, though, because you can't scan through memcached keys, you can only look them up, so we shouldn't have to worry about conflicts.